src/rideshare/settings.py

```diff
@@ -26,8 +26,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = environ.get("DJANGO_DEBUG")
 
-ALLOWED_HOSTS = []
-
+ALLOWED_HOSTS = environ.get("DJANGO_ALLOWED_HOSTS", "").split(" ")
```

Comment:
did we figure out the issue with this environment variable not being recognized?

Reply:
i'm guessing we had to include the default "" value as the second argument or else it would register as null? it seems to work now tho